config/environments/development.rb

```diff
@@ -1,23 +1,23 @@
 Rails.application.configure do
-    # Verifies that versions and hashed value of the package contents in the project's package.json
-  config.webpacker.check_yarn_integrity = true
-
   config.active_job.queue_adapter = :async
+  config.action_mailer.delivery_method = :letter_opener
   config.cache_classes = false
   config.eager_load = false
   config.consider_all_requests_local = true
-  if Rails.root.join('tmp/caching-dev.txt').exist?
+  if Rails.root.join('tmp', 'caching-dev.txt').exist?
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Rails/FilePath: Please use Rails.root.join('path/to') instead.